bootstrap.json

```diff
@@ -1,7 +1,6 @@
 {
     "after-scripts": [
-        "brew linkapps",
```

Comment:
this is used to link the installed apps to the /Applications folder. i think we should keep this.